samples/v1/language_sentiment_gcs.py

```diff
@@ -43,7 +41,7 @@ def sample_analyze_sentiment(gcs_content_uri):
     # gcs_content_uri = 'gs://cloud-samples-data/language/sentiment-positive.txt'
 
     # Available types: PLAIN_TEXT, HTML
-    type_ = enums.Document.Type.PLAIN_TEXT
+    type_ = language_v1.enums.Document.Type.PLAIN_TEXT
```

Comment:
There isn't a generated `enums` submodule any more. Does the sample run?

Reply:
My bad. Fixed now. Moreover these samples (in samples/v1/) are pre-existing ones and not from python-doc-samples repo.